Rubberduck.VBEEditor/ComManagement/TypeLibs/TypeApiFactory.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Runtime.InteropServices.ComTypes;
+using Rubberduck.VBEditor.ComManagement.TypeLibs.Abstract;
+
+// TODO The tracers are broken - using them will cause a NRE inside the 
+// unmanaged boundary. If we need to enable them for diagnostics, this needs
+// to be fixed first. 
+#if DEBUG && TRACE_TYPEAPI
+using Rubberduck.VBEditor.ComManagement.TypeLibs.DebugInternal;
+#endif
```

Comment:
No biggie, but unused `using` statements are simply ignored by the compiler, there's no need to conditionally-compile them.

Reply:
1) there's also the `TRACE_TYPEAPI` which needs to be manually enabled -- `DEBUG` should not be sufficient on its own. 

2) I have in mind to do a separate PR to get rid of all those fugly conditional compilations for [reasons described in this blog](https://blogs.msmvps.com/peterritchie/2011/11/24/if-you-re-using-if-debug-you-re-doing-it-wrong/). However, there are number of them (even predating this PR), so it has to be its own PR.